src/rateEngine/__tests__/rates/data/TOU_EV_7.ts

```diff
@@ -0,0 +1,23 @@
+import touEV7RateElements from '../../../__mocks__/rates/tou-ev-7.json';
+
+// https://www.sce.com/business/rates/electric-car-business-rates/business/rates/electric-car-business-rates
+// https://www.sce.com/sites/default/files/inline-files/TOU-EV-7_8_9%20Rate%20Fact%20Sheet_WCAG%20(2).pdf
+// https://edisonintl.sharepoint.com/teams/Public/TM2/Shared%20Documents/Forms/AllItems.aspx?id=%2Fteams%2FPublic%2FTM2%2FShared%20Documents%2FPublic%2FRegulatory%2FTariff%2DSCE%20Tariff%20Books%2FElectric%2FSchedules%2FGeneral%20Service%20%26%20Industrial%20Rates&p=true&ga=1
+//
+// Service under this Schedule will be supplied at one standard voltage.
+const touEV7 = (year: number) => ({
```

Comment:
could we get rid of the year altogether in these tests? the "multiple years" aspect of these rate definitions isn't really part of the rate engine's scope, and we're only using a single year in the tests anyway. would remove a bunch of code, make the tests easier to grok for Future New Developer, and doesn't affect the power of the tests at all (we're still testing the exact same data/behavior inside the rate engine)